src/Microsoft.Framework.CodeGeneration.Common/TemplateFoldersUtilities.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#if DNX451 || DNXCORE50
```

Comment:
This is used by Microsoft.Framework.CodeGeneration.EntityFramework and .Mvc. But not by .Templating. So updated so it only applies there.